docs/guides/python.md

```diff
@@ -0,0 +1,163 @@
+# Python Developer's Guide
+
+When creating a Python function using the `func` CLI, the project directory
+looks like a typical Python project. Both HTTP and Event functions have the same
+template structure.
+
+```
+~/src/fn
+❯ kn func create -l python
```

Comment:
The rest of the Guide (as well as the others) shows usage of `func` not `kn func`. Would be nice to make it consistent.
```suggestion
❯ func create -l python fn
```